src/components/textarea/index.md.njk

```diff
@@ -41,6 +45,11 @@ Make the height of a textarea proportional to the amount of text you expect user
 
 Users will often need to copy and paste information into a textarea, so you shouldn’t prevent them from doing this.
 
+### Limiting the number of characters
+
+If there’s a good reason to limit the number of characters users can enter, you can use the [character count](https://design-system.service.gov.uk/components/character-count/) component.
```

Comment:
Hey Andrew!

This is looking good, I've noticed you have used an absolute link where we need to use a relative one like you have above.

If you want to make this change feel free, otherwise we can do this for you.

Let me know.